instrumentation/kafka/README.md

```diff
@@ -0,0 +1,26 @@
+# Brave Kafka instrumentation
+
+Add decorators for Kafka producer and consumer to enable tracing for Single Producer/Single Consumer (for now).
+The producer propagates the B3 headers in the records headers.
+The consumer is responsible to close the producer span.
+
+To use the producer simply wrap it like this : 
+```java
+Producer<String, String> stringProducer = new KafkaProducer<>(settings);
+TracingProducer<String, String> tracingProducer = new TracingProducer<>(tracing, mockProducer);

```

Comment:
Could be any kind of producer. You made me realize that I wrongly copy/pasted this code for the documentation.
Will change to ```realProducer``` and remove the types.